next.config.js

```diff
@@ -99,4 +99,16 @@ const sentryWebpackPluginOptions = {
   },
 };
 
-module.exports = withBundleAnalyzer(withSentryConfig(nextConfig, sentryWebpackPluginOptions));
\ No newline at end of file
+module.exports = {
+  ...withBundleAnalyzer(withSentryConfig(nextConfig, sentryWebpackPluginOptions)),
+  images: {
+    remotePatterns: [
+      {
+        protocol: 'https',
+        hostname: 's2.coinmarketcap.com',
+        port: '',
+        pathname: '/**',
+      },
+    ],
+  },
```

Comment:
We shouldn't allow this. The Immunefi report from a few weeks ago showed that maliciously crafted SVGs on other domains + Next's custom `Image` component can be easily exploited to hijack the page content and phish users. The fix was to lock down this image.remotePatterns setting and we shouldn't open it back up

Reply:
Yeah very happy to not commit this - this was just a PoC we hacked together to show a partner what the experience would look like